commands/trade.js

```diff
@@ -114,11 +114,10 @@ module.exports = function container (get, set, clear) {
               limit: 1000
             }
             if (db_cursor) {
-              trade_cursor = db_cursor
               opts.query.time = {$gt: db_cursor}
             }
             else {
-              trade_cursor = query_start
```

Comment:
This line however is necessary, since we may quit pre-roll after 1 iteration and no data may be present. In that case we would have an invalid trade_cursor once we started updating the engine causing exchange.getTrades to return and process as many results as possible in a single API call.